asetukset.edn

```diff
@@ -146,9 +146,9 @@
                         :kanavien-osoite "https://avoinapi.vaylapilvi.fi/vaylatiedot/wfs?request=GetFeature&service=WFS&version=1.1.0&typeName=vesivaylatiedot:sulkukanavat&outputFormat=SHAPE-ZIP"
                         :kanavien-tuontikohde "./shp/Vesivaylien_kanavat/SulkukanavatPolygon.shz"
 
-                        :tieturvallisuustarkastus-tieverkko-shapefile "file://shp/Tieturvallisuus/tieturvallisuustarkastustiesto.shp"
-                        :tieturvallisuustarkastus-tieverkko-osoite "https://avoinapi.vaylapilvi.fi/vaylatiedot/wfs?request=GetFeature&service=WFS&version=1.1.0&typeName=tiestotiedot:tieturvallisuustieverkko&outputFormat=SHAPE-ZIP"
-                        :tieturvallisuustarkastus-tieverkko-tuontikohde "./shp/Tieturvallisuus/tieturvallisuustarkastustiesto.shz"
+                        :tieturvallisuustarkastus-tieverkko-shapefile "file://shp/Tieturvallisuus/tieturvallisuustarkastettavat_tietLine.shp"
```

Comment:
Onko näitä muutoksia tehty uudempaan harja-infra repoon ja harja-ci repoon?

Reply:
Muutokset on tehty muihin repoihin maanantaina